### PR TITLE
COTECH-581 | Use hasAds in standalone player

### PR DESCRIPTION
**Pull request by latata**
https://fandom.atlassian.net/browse/COTECH-581

### CODE REVIEW

File: stand-alone/standalone-loader.tsx [hunk 1]
```diff
@@ -80,6 +80,7 @@ window.loadPlayer = async (context: RedVenturePlayerContextProps) => {
 			jwPlayerContainerEmbedId: getJwPlayerContainerEmbedId(context),
 			playerUrl: context?.playerUrl,
 			autoPlay: context?.autoPlay,
+			hasAds: context.hasAds,
```
Comment (andrzejkupczyk) [inline stand-alone/standalone-loader.tsx:83]: Based on `RedVenturePlayerContext` interface, this parameter is optional and could potentially be undefined.

```suggestion
			hasAds: context?.hasAds,
```

File: src/jwplayer/types.d.ts [hunk 2]
```diff
@@ -486,6 +486,7 @@ export interface RedVentureVideoPlayerProps extends JwPlayerContainerId {
 	showScrollPlayer: boolean;
 	playerUrl?: string;
 	autoPlay?: boolean;
+	hasAds?: boolean;
```
Comment (andrzejkupczyk) [inline src/jwplayer/types.d.ts:489]: Should we also update `stand-alone/types.d.ts`?

File: stand-alone/standalone-loader.tsx [hunk 3]
```diff
@@ -80,6 +80,7 @@ window.loadPlayer = async (context: RedVenturePlayerContextProps) => {
...
 		} as RedVentureVideoPlayerProps),
```
Comment (andrzejkupczyk) [inline stand-alone/standalone-loader.tsx:84]: It seems that the conversion of type `{ videoDetails: RedVentureVideoDetails, ... }` to type `RedVentureVideoPlayerProps` could be a mistake, as these types are not compatible. I suspect that this is intentional. However, should we convert the expression to `unknown` first?

```ts
videoDetails: redVentureVideoDetails as unknown,
```

Currently, it should throw the TS2352 error 🤔

File: stand-alone/standalone-loader.tsx (hunk 1 above)
Comment (latata) [reply stand-alone/standalone-loader.tsx:83]: The `hasAds` is optional but `context` is not. I think these `?` above are redundant.

File: stand-alone/standalone-loader.tsx (hunk 3 above)
Comment (latata) [reply stand-alone/standalone-loader.tsx:84]: As that issue was already there I'd not bother as it's not critical (no runtime issue just types) and we are most likely sunsetting jwplayer-fandom soon :)

Comment (latata) [reply stand-alone/standalone-loader.tsx:84]: ☝️ that is actually not true -adding `hasAds?: boolean;` to `stand-alone/types.d.ts` resolves the issue. I will add it

Comment (andrzejkupczyk) [reply stand-alone/standalone-loader.tsx:84]: The error TS2352 has been resolved. Thank you very much!

File: src/jwplayer/types.d.ts (hunk 2 above)
Comment (andrzejkupczyk) [reply src/jwplayer/types.d.ts:489]: This is no longer applicable 👍🏻

File: stand-alone/standalone-loader.tsx (hunk 1 above)
Comment (andrzejkupczyk) [reply stand-alone/standalone-loader.tsx:83]: Thanks for the clarification. You could be right, sir. However, something still seems a bit off to me 🤔 I might have misunderstood the changes, but shouldn't its value default to true? If it's undefined, that wouldn't happen, would it? 😬

Comment (andrzejkupczyk) [reply stand-alone/standalone-loader.tsx:83]: Discussed directly 👍🏻

**Pull request merged**